app/src/main/java/com/nextcloud/talk/adapters/items/ConversationItem.java

```diff
@@ -108,7 +108,7 @@ public Conversation getModel() {
 
     @Override
     public int hashCode() {
-        return conversation.hashCode();
+        return conversation.hashCode() * (status == null ? 1 : status.hashCode());
```

Comment:
Multiplying both hashcodes can result in overflow. 

Here's how the autogenerator does it instead: multiply one of them by a prime number and sum them

```suggestion
        int result = conversation.hashCode();
        result = 31 * result + (status != null ? status.hashCode() : 0);
        return result;
```

Reply:
👍